packages/forms-web-app/src/controllers/appellant-submission/submission.js

```diff
@@ -23,6 +25,7 @@ exports.postSubmission = async (req, res) => {
       const { appeal } = req.session;
       appeal.state = 'SUBMITTED';
       req.session.appeal = await createOrUpdateAppeal(appeal);
+      await storePdfAppeal(appeal);
     } catch (e) {
       logger.error(e);
```

Comment:
Simon's meeting on Friday afternoon was about moving to `req.logger.{method}`, as that way we can track an error from start to finish using the correlation id.